src/components/ScrollTable/ScrollTable copy.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { createClass, omitProps } from '../../util/component-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import Table from '../Table/Table';
+
+const cx = lucidClassNames.bind('&-ScrollTable');
+
+const { object, string, bool, node, number, oneOfType } = PropTypes;
+
+const ScrollTable = createClass({
```

Comment:
did you mean to include this file? `ScrollTable copy.jsx`

Reply:
Good eye. Nope. That will come out.

Reply:
Nope. Good catch.